Sources/TuistLoader/Models+ManifestMappers/Workspace+ManifestMapper.swift

```diff
@@ -19,7 +19,6 @@ extension XcodeGraph.Workspace {
         func globProjects(_ path: Path) throws -> [AbsolutePath] {
             let resolvedPath = try generatorPaths.resolve(path: path)
             let projects = FileHandler.shared.glob(AbsolutePath.root, glob: String(resolvedPath.pathString.dropFirst()))
-                .lazy
```

Comment:
this is the actual fix. When using `lazy`, the filters below are skipped. Since the operations are fast, I don't think this should cause a performance regression